packages/hub/package.json

```diff
@@ -21,6 +21,7 @@
     "@cardstack/di": "^0.3.4",
     "@cardstack/elasticsearch": "^0.4.1",
     "@cardstack/handlebars": "^0.3.4",
+    "@cardstack/logger": "^0.1.0",
```

Comment:
I think every package that instantiates a logger needs to do this directly.

Reply:
We have a gap right now in our testing that lets this class of bug slip through -- packages can depend on each other's dependencies without declaring them properly, thanks to yarn workspaces. I think we can plug the gap via eslint.